index.html

```diff
@@ -30,8 +44,32 @@ <h1 class="title-bar">Awesome books</h1>
             </form>
         </article>
     </div>
-</main>
-  </div>
-  
+    <section class="contact">
+      <h2 class="contact__header">Contact Information</h2>
+      <p class="contact__text">If you have any questions or what to say "Hello" <br> You can reach us on:</p> 
+      <div>    
+      <ul class="contact__info">
+          <li>jonathan</li>
+          <li>
+            <a href="https://github.com/KAYEMBAJONATHAN">mybio</a>
+          </li>
+          <li>Pretoria, South Africa</li>
+        </ul>
+        <br>
+        <ul class="contact__info">
+          <li>HENARY</li>
+          <li>
+            <a href="https://github.com/HenaryKhuraijam">mybio</a>
+          </li>
+          <li>Zambia, Lusaka</li>
+        </ul>
+      </div>
+    </section>
+  </main>
```

Comment:
- [ ] Right now I don't see any content on the main section. and when I click on any link, it doesn't show me the relevant section. This issue was also mentioned by the previous reviewer, kindly consider implementing that.
kindly have a close look at [the project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website_v1_1.md) and try to implement them thoroughly.
your project 

https://user-images.githubusercontent.com/96683843/228631885-cdd4ea61-9127-4555-a105-668eced41d27.mov